internal/channelz/logging.go

```diff
@@ -21,22 +21,24 @@ package channelz
 import (
 	"fmt"
 
-	"google.golang.org/grpc/internal/grpclog"
+	"google.golang.org/grpc/grpclog"
```

Comment:
What was the difference between `internal/grpclog` and `grpclog`?
Is it OK to simply use component where `internal/grpclog` was used?

Reply:
channelz/logging use to use the depth logging interface (InfoDepth etc.) which is in internal. Now it uses the component InfoDepth interface which is just in grpclog. Anywhere where to want to depth log without using a component would still need internal, otherwise with normal grpclog logging (including through components) they should use the non-internal.